src/screens/OrgList/OrgList.tsx

```diff
@@ -284,15 +300,17 @@ function OrgList(): JSX.Element {
                 }}
               >
                 <tbody>
-                  <tr data-testid="rowsPPSelect">
-                    <PaginationList
-                      count={data ? data.organizationsConnection.length : 0}
-                      rowsPerPage={rowsPerPage}
-                      page={page}
-                      onPageChange={handleChangePage}
-                      onRowsPerPageChange={handleChangeRowsPerPage}
-                    />
-                  </tr>
+                  {data_2?.user.userType !== 'ADMIN' && (
```

Comment:
If this section is only for super admins why not use a conditional that represents that logic. Using !ADMIN implies there are other user types that can access this part of the UI. Is that so?